WMF Framework/Router.swift

```diff
@@ -69,6 +76,15 @@ public class Router: NSObject {
             return project.supportsNativeUserTalkPages ? .userTalk(url) : nil
         case .special:
             
+            if title == "ReadingLists",
```

Comment:
More of a curiosity, will this behave like other namespaces (that are language-specific)?

Reply:
Good eye - I think line 65  above (`namespaceAndTitleOfWikiResourcePath`) should be flexible enough to handle the translated "Special" namespace, so if links come through as `es.wikipedia.org/wiki/Especial:ReadingLists?limport=nnn` it should still flow through this router correctly. I haven't tested a full end-to-end flow on a non-EN deep link yet and was planning on waiting till the landing page was in production first.

With "ReadingLists" being the title - I am unsure if that will stay as this translation or not - I'll check with Android. This (and probably other areas) will break if not.

Reply:
Update - we can expect the "ReadingLists" title to remain static for all wikis.